QUARKUS-6245.md

```diff
@@ -0,0 +1,27 @@
+# QUARKUS-6245 Allow setting Clear-Site-Data on OIDC logout
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6245
+
+Upstream PR: https://github.com/quarkusio/quarkus/pull/46864
+
+Upstream documentation:
+* https://quarkus.io/guides/security-oidc-code-flow-authentication#logout-and-expiration
+
+This feature allows to set [Clear-Site-Data](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Clear-Site-Data) http header on logout from OIDC.
```

Comment:
```suggestion
This feature allows to set [Clear-Site-Data](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Clear-Site-Data) HTTP header on logout from OIDC.
```

Reply:
Makes sense @mocenas . I don't know how the clear-side data fits infor the backchannel logout, but I think I saw it in the code so it is happening.